events/wc_dark_cult_events.txt

```diff
@@ -606,7 +606,7 @@ province_event = {
 
 	immediate = {
 		# Starts "The Voice from Beyond the Sea" event chain
-		province_event = { id = WCDAC.5029 days = 1825 }
+		province_event = { id = WCDAC.5029 years = 6 }
```

Comment:
Decrease this time at least for testing purposes, it's painful to wait.

Reply:
@sejtalk, done in 2d86e0f5692aa89a0198ee5527c8f31a238a7980. If you want, revert it when it's ready to merge.